py/torch_tensorrt/dynamo/conversion/_TRTInterpreter.py

```diff
@@ -256,6 +256,10 @@ def run_node(self, n: torch.fx.Node) -> torch.fx.Node:
         n.kwargs = kwargs
 
         # run the node
+        _LOGGER.debug(
```

Comment:
Can we pull out some more metadata here, like input names and types, output shape etc.?